NOTES.rst

```diff
@@ -8,6 +8,16 @@ existing bug reports, go to https://github.com/uninett/nav/issues .
 To see an overview of upcoming release milestones and the issues they resolve,
 please go to https://github.com/uninett/nav/milestones .
 
+NAV 5.4
```

Comment:
you should but `(not yet released)` after the title here, so as not to confuse anyone into believing this is an actual release yet. We usually remove it once we actually do create a `5.4.x` series branch.